test/integration/integration_test.go

```diff
@@ -156,7 +112,7 @@ func getLogLevel() (level int) {
 // setup sets up integration tests
 func (i *integrationTest) setup() {
 	var err error
-	i.engineClient, err = dockerClient(dockerConfig())
+	i.engineClient, err = dockerapi.NewEnvClient()
```

Comment:
On the Mac, this is not enough. A bug in the engine API client sets the default docker host to tcp://127.0.0.1:2375, instead of unix:///var/run/docker.sock. You need a workaround like this:
https://github.com/openshift/origin/blob/master/pkg/bootstrap/docker/up.go#L437-L442
(before invoking NewEnvClient)